### PR TITLE
first push

### PR DESCRIPTION
**Pull request by raman-cn**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+
+ENV PYTHONUNBUFFERED 1
+
+WORKDIR /Downloads/assign1
+
+COPY ./requirements.txt /app/requirements.txt
+
+RUN pip install -r /app/requirements.txt
+
+ADD . /Downloads/assign1
```
Comment (iamKunalGupta) [inline Dockerfile:11]: Why are you using this directory `/Downloads/assign1`?

Comment (raman-cn) [reply Dockerfile:11]: My laptop was not working properly,So, I have to reinstall Ubuntu yesterday while creating the project. So I have made the zip save it in my google drive and then downloaded that from their!!
So used the same Directory!

Comment (iamKunalGupta) [reply Dockerfile:11]: Yes, but why the `WORKDIR`?
Also do see the other comment as well.

Comment (raman-cn) [reply Dockerfile:11]: I though we have to make the work directory, the directory where we have our whole project

Comment (iamKunalGupta) [reply Dockerfile:11]: @raman-cn `WORKDIR` is for a directory inside the Docker image/container. You didn't have to set is as what you did. The context you pass while building the image matters.

Comment (raman-cn) [reply Dockerfile:11]: I have pushed the hpa and service specs too!!

Comment (raman-cn) [reply Dockerfile:11]: I will learn about WORKDIR and make sure that it won't happen again!

Comment (raman-cn) [reply Dockerfile:11]: with any other file or any other command

Comment (iamKunalGupta) [reply Dockerfile:11]: sure np

File: assign1/settings.py [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+
+# Build paths inside the project like this: os.path.join(BASE_DIR, ...)
+BASE_DIR = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
+
+
+# Quick-start development settings - unsuitable for production
+# See https://docs.djangoproject.com/en/3.0/howto/deployment/checklist/
+
+# SECURITY WARNING: keep the secret key used in production secret!
+SECRET_KEY = 'w!$b$$ia9)7t*#)hzmy6@9=_c_3jz9ukr$nmog_!c@rcp3nqqf'
```
Comment (lohitcn) [inline assign1/settings.py:23]: Don't push secret values.
Use environmnet variables or load secrets from a local JSON/YAML file and upload the template of JSON/YAML here.

**Pull request merged**